src/components/Header.svelte

```diff
@@ -14,6 +15,7 @@
     <a href={`${base}/#about`}><Localized id="header--about" /></a>
     <a href={`${base}/cfp`}><Localized id="header--cfp" /></a>
     <LanguageSwitcher />
+    <ThemeSwitcher />
```

Comment:
The header has too many elements now, and the `Home` item is overlapped when in a small resolution (e.g., on an iPhone).
We should replace `Home / Sobre / CFP` with a dropdown when using a small resolution.
![telegram-cloud-photo-size-1-4963464739736890066-y](https://user-images.githubusercontent.com/9501115/227132238-ad4c95aa-2634-4347-bf3a-7c4634dd0681.jpg)

Reply:
Okay, we can postpone it